drivers/gpio/hcsr04_driver.go

```diff
@@ -0,0 +1,166 @@
+package gpio
+
+import (
+	"errors"
+	"fmt"
+	"sync"
+	"time"
+
+	"gobot.io/x/gobot"
+	"gobot.io/x/gobot/drivers/gpio"
+)
+
+const (
+	soundSpeed float32 = 343.0 // in [m/s]
+	// the device can measure 2cm .. 4m, this means sweep distances between 4cm and 8m
+	// this cause pulse durations between 0.12ms and  24ms (at 34.3 cm/ms, ~0.03 ms/cm, ~3ms/m)
+	measurementCycle time.Duration = 60 // 60ms between two measurements
+	MonitorUpdate    time.Duration = 100 * time.Millisecond
+)
+
+// HCSR04 instance
+type HCSR04 struct {
+	name                   string
+	connection             gobot.Adaptor
+	triggerPin             *gpio.DirectPinDriver

```

Comment:
Because the file is already part of package "gpio", all this "gpio." needs to be removed. Otherwise an import cycle failure occurs.